### PR TITLE
Add all possible mutations for strings, numbers, arrays, maps.

### PR DESCRIPTION
**Pull request by judepereira**

### CODE REVIEW

File: storyscript/README.md [hunk 1]
```diff
@@ -161,24 +161,51 @@ tweet like     # calls "like", a method of tweet
...
+num is_even  # returns true
+num absolute  # returns the absolute value (if num was -1, this would return 1)
+num increment  # returns 11. Note that num is not changed
+num decrement  # returns 9. Note that num is not changed
+
+# Arrays
+arr = [1, 2, 3, 4, 5]
+arr index of: 5  # returns the index of an element, 4 in this case
+arr length  # returns the length of the array, 5 in this case
+arr append element: 6  # adds 6 to the end of the array
```
Comment (stevepeak) [inline storyscript/README.md]: `item`? Element feels long and strange IMO

Comment (judepereira) [reply storyscript/README.md]: Oh good catch, yes, item is better and consistent :)

**Comment by stevepeak** [comment]
@judepereira What about applying methods to a list? `myList apply method:string.capitalize` for example. Equivalent to Python's map `map(capitalize, myList)`

**Comment by judepereira** [comment]
PS - that's a ugly looking one liner :(

**Pull request merged**